.gitlab-ci.yml

```diff
@@ -234,7 +234,7 @@ deploy-staging:
     - .staging
   script:
     - yarn
-    - BUILD_MODE=staging yarn build:bundle
+    - BUILD_MODE=canary yarn build:bundle
     - ./scripts/deploy.sh staging staging
```

Comment:
Should we use the `canary` suffix on staging env then?
or is it still useful to differentiate canary and staging files?

Reply:
Yes it is useful as they are not build at the same time / frequency.
- canary on commit on main
- staging on commit on staging

Reply:
ok, fair enough